browser/resources/settings/brave_sync_page/brave_sync_subpage.ts

```diff
@@ -187,6 +199,17 @@ class SettingBraveSyncSubpage extends SettingBraveSyncSubpageBase {
   */
   computeHasSyncWordsDecryptionError_() {
     return this.syncStatus != undefined && !!this.syncStatus.hasSyncWordsDecryptionError
+      && !this.syncStatus.isOsEncryptionAvailable
+  }
+
+  /**
+  * @return {boolean}
+  * @private
+  */
+  computeHasSyncWordsDecryptionErrorUnlockedSs_() {
```

Comment:
you are right, I chose `computeHasSyncWordsDecryptionErrorUnlockedSs_` because `computeHasSyncWordsDecryptionErrorUnlockedSafeStorage_` was extremely long.
I will rename it.

Reply:
thanks, renamed at https://github.com/brave/brave-core/pull/16117/commits/5e46254c821d56baaa48b15e09b1135ef9dc118f